src/dataexplorer/dataexplorer-application/src/components/QuestionnaireResponse/SchemaResponse/SchemaResponse.tsx

```diff
@@ -103,27 +98,17 @@ const SchemaResponse = (props: { questionnaireResponseId: string }) => {
 
     return (
         <>
-            {schemaResponse && questionnaire && questionnaire.name && (
+            {questionnaire && questionnaire.name && (
                 <div>
-                    <Schemes
+                    <SchemaView
                         questions={questions}
                         answers={answers}
                         questionnaireResource={questionnaireResource}
-                        date={dayjs(schemaResponse.response?.authored)}
+                        date={dayjs(props.schemaResponse.authored)}
                         title={questionnaire.name}
                     />
                 </div>
             )}
-            {!schemaResponse.response && schemaResponse.error.length === 0 && (
-                <Row justify="space-around" align="middle">
-                    <Spin size="large" />
-                </Row>
-            )}
-            {schemaResponse.error.length > 0 && (
-                <Row justify="center">
-                    Feil ved lasting av skjema: {schemaResponse.error}
-                </Row>
-            )}
```

Comment:
Any particular reason this was removed?

Reply:
It's moved to QuestionnaireResponse.tsx instead now, since the questionnaireResponse fetching happens there